packages/react-jsx-highcharts-gantt/src/components/Pathfinder/Pathfinder.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { attempt } from 'lodash-es';
+import {getModifiedProps} from 'react-jsx-highcharts';
+
+class Pathfinder extends Component {
+
+  static propTypes = {
+    getChart: PropTypes.func.isRequired, // Provided by ChartProvider
+    enabled: PropTypes.bool.isRequired,
+    dashStyle: PropTypes.string(),
+    lineWidth: PropTypes.number()
+  };
+
+  static defaultProps = {
+    enabled: true
+  };
+
+  constructor (props) {
+    super(props);
+
+    this.state = {
+      rendered: false
+    };
```

Comment:
This state isn't used anywhere, so this constructor is unnecessary

Reply:
Sure, copy and paste.